RELEASING.md

```diff
@@ -131,7 +131,7 @@ Here are the steps:
    - `adoptBuildNumber`: Leave blank unless you are doing a point release in which case it should be a number starting at `1` for the first point release.
    - `additionalConfigureArgs`: JDK8 automatically adds`--with-milestone=fcs` in `build.sh` so there's no need to provide it here. For JDK11+ use `--without-version-pre --without-version-opt` (for EA releases use: `--with-version-pre=ea --without-version-opt`)
    - `scmReference`: One of the following:
-     - For HotSpot, it's the same tag suffixed with `_adopt` e.g. `jdk-13.0.1+9_adopt`
+     - For HotSpot, it's the same tag suffixed with `_adopt`(for old builds) or `_adoptium` e.g. `jdk-13.0.1+9_adopt`
```

Comment:
I would swap the order to avoid confusion:

```suggestion
     - For HotSpot, it's the same tag suffixed with `_adoptium` or `_adopt`(for old builds) e.g. `jdk-17.0.2+9_adoptium`
```